python/semantic_kernel/agents/open_ai/responses_agent_thread_actions.py

```diff
@@ -311,11 +321,16 @@ async def invoke_stream(
             temperature: The temperature.
             top_p: The top p.
             truncation: The truncation strategy.
+            verbosity: The verbosity level (only supported by GPT-5 family models) controlling how much internal
+                model reasoning is surfaced in the natural language response ("low", "medium", or "high").
             kwargs: Additional keyword arguments.
 
         Returns:
             An async iterable of tuple of the visibility of the message and the chat message content.
         """
+        # Validate reasoning effort parameter
+        cls._validate_reasoning_effort_parameter(reasoning)
```

Comment:
I don't think we need to be so defensive with this validation check.

Reply:
removed